src/components/product-details/place-bet.tsx

```diff
@@ -7,24 +7,25 @@ import Button from '@ui/button';
 import PlaceBidModal from '@components/modals/placebid-modal';
 import Countdown from '@ui/countdown/count-down';
 import { ImageType } from '@utils/types';
-import { HighestBid } from '@types';
+import { HighestBid, Item } from '@types';
 
 type Props = {
-  highestBid: HighestBid;
+  highestBid?: HighestBid;
   auctionDate?: string;
   btnColor?: 'primary' | 'primary-alta';
   className?: string;
+  item: Item;
 };
 
-const PlaceBet = ({ highestBid, auctionDate, btnColor, className }: Props) => {
+const PlaceBet = ({ highestBid, auctionDate, btnColor, className, item }: Props) => {
   const [showBidModal, setShowBidModal] = useState(false);
   const handleBidModal = () => {
     setShowBidModal((prev) => !prev);
   };
   return (
     <>
       <div className={clsx('place-bet-area', className)}>
-        <div className="rn-bet-create">
+        {/* <div className="rn-bet-create">
```

Comment:
Same here, if you think we won't use this chunk of code in a future, is better to remove it